mne/io/reference.py

```diff
@@ -97,11 +128,16 @@ def _apply_reference(inst, ref_from, ref_to=None):
                 'using the apply_proj() method function before the desired '
                 'reference can be set.'
             )
+
     for i in projs_to_remove:
         del inst.info['projs'][i]
```

Comment:
it seems obvious though that if you first apply an avg ref, and then start applying some different ref, that the resulting signal is no longer in avg ref. Who would assume otherwise?